caffe2/python/operator_test/weighted_sum_test.py

```diff
@@ -4,20 +4,22 @@
 from __future__ import unicode_literals
 
 from caffe2.python import core
-from hypothesis import given
 import caffe2.python.hypothesis_test_util as hu
 import hypothesis.strategies as st
 import numpy as np
+import caffe2.python.serialized_test.serialized_test_util as serial
```

Comment:
nit: Could move this up after `caffe2.python.hypothesis_test_util` to be sorted.